### PR TITLE
tagged vlan read by minigraph

### PR DESCRIPTION
**Pull request by kktheballer**
**- Why I did it**
To enable tagged vlan support in SONiC

**- How I did it**
Made a minor change to minigraph parser to distinguish between tagged and untagged vlans

**- How to verify it**
Test minigraph parser on a TOR by doing a reload of a  minigraph containing tagged vlans and ensure the correct configuration of the vlans on the device

**- Description for the changelog**
 

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -340,11 +340,15 @@ def parse_dpg(dpg, hname):
             vintfname = vintf.find(str(QName(ns, "Name"))).text
             vlanid = vintf.find(str(QName(ns, "VlanID"))).text
             vintfmbr = vintf.find(str(QName(ns, "AttachTo"))).text
+            vlantype = vintf.find(str(QName(ns, "Type"))).text
             vmbr_list = vintfmbr.split(';')
             for i, member in enumerate(vmbr_list):
                 vmbr_list[i] = port_alias_map.get(member, member)
                 sonic_vlan_member_name = "Vlan%s" % (vlanid)
-                vlan_members[(sonic_vlan_member_name, vmbr_list[i])] = {'tagging_mode': 'untagged'}
+                if vlantype == "UserDefinedL2Vlan":
```
Comment (lguohan) [inline src/sonic-config-engine/minigraph.py:348]: what is UserDefinedL2Vlan?

**Comment by lguohan** [comment]
please also add unit test for minigraph

File: src/sonic-config-engine/minigraph.py (hunk 1 above)
Comment (kktheballer) [reply src/sonic-config-engine/minigraph.py:348]: Thanks for reviewing! This is just a keyword we have created internally within our team to denote a tagged vlan type. Hopefully this is an ok naming convention?

Comment (kktheballer) [reply src/sonic-config-engine/minigraph.py:348]: I have also added the UT. Please let me know if it looks ok.

**Comment by lguohan** [comment]
superseded by #4731

**Pull request closed without merging**